scripts/babel-register.js

```diff
@@ -1,10 +0,0 @@
-"use strict";
-
-const register = require("babel-register").default;
-
-register({
-  extensions: [".js"],
-  // Only js files in the test folder but not in the subfolder fixtures.
-  only: [/packages\/.+\/test\/(?!fixtures\/).+\.js$/],
-  compact: true,
```

Comment:
Removed this since it is kind of annoying when debugging with `--inspect` in tests.